### DISCUSSION TITLE
Way to get fields that are relations in a query in Prisma 2?

### DISCUSSION THREAD

**Discussion by MaxwellGover** [prisma/prisma#2446] [Q&A] [answered]
Say I have a schema that looks like this

```
model Song {
  id        Int      @default(autoincrement()) @id
  createdAt DateTime @default(now())
  title     String
  url       String
  user      User     @relation(fields: [userId], references: [id])
  userId    Int
}

model User {
  id Int @id @default(autoincrement())
  email String @unique
  username String @unique
  displayName String
  password String
  songs Song[]
  role Role @default(USER)
  resetToken String?
  resetTokenExpiry Float?
  createdAt DateTime @default(now())
}
```
How would I go about making songs available on the `User` type? I know in Prisma 1 you had to do something involving fragments but I thought I remembered seeing that they were going to clean this up in Prisma 2 in favor of another way. 

```
const fragment = `
      fragment UserWithUploads on User {
        id
        songs {
          id
          title
          url
        }
      }
    ` 
```

**Comment by ryands17** [comment, marked as answer]
Hey @MaxwellGover :wave: 
Currently you can write the following the get the user and their songs:

```ts
await prisma.user.findOne({
  where: { id: <some-id> },
  include: {
    songs: true,
  },
})
```

This will give you the user with all their songs You can learn more about this in the [docs](https://www.prisma.io/) as well.

**Reply by MaxwellGover** [reply]
Thanks!

**Comment by ludralph** [comment]
Hi @MaxwellGover,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
